tools/IntegrationsJsonGenerator/Program.cs

```diff
@@ -22,6 +22,7 @@
 using System.Runtime.CompilerServices;
 using System.Text.Encodings.Web;
 using System.Text.Json;
+using IntegrationsJsonGenerator;
```

Comment:
I suggest adding `<ImplicitUsings>enable</ImplicitUsings>` in `csproj`.

Reply:
#1892